cargo/cargo_environ.bzl

```diff
@@ -0,0 +1,28 @@
+"""Helper functions for generating the environment for cargo executions."""
+
+load("//cargo/private:cargo_utils.bzl", "cargo_home_path")
+
+CARGO_BAZEL_ISOLATED = "CARGO_BAZEL_ISOLATED"
```

Comment:
This is unique to `crate_universe` and should probably stay within that package.

Reply:
It no longer is unique to crate_universe after this change. Would the following work?

* Introduce a new `CARGO_ISOLATED` env var and use it int he cargo directory
* Make `CARGO_BAZEL_ISOLATED` default to the value of `CARGO_ISOLATED`